crates/biome_cli/src/execute/traverse.rs

```diff
@@ -567,12 +568,19 @@ fn process_messages(options: ProcessMessagesOptions) {
         }
     }
 
-    for diagnostic in diagnostics_to_print {
+    for diagnostic in &diagnostics_to_print {
         console.error(markup! {
-            {if verbose { PrintDiagnostic::verbose(&diagnostic) } else { PrintDiagnostic::simple(&diagnostic) }}
+            {if verbose { PrintDiagnostic::verbose(diagnostic) } else { PrintDiagnostic::simple(diagnostic) }}
         });
     }
 
+    let emit_gh_diags = mode.is_ci() && mode.should_report_to_terminal() && mode.is_running_in_ci();
+    if emit_gh_diags {
+        for diagnostic in diagnostics_to_print {
+            console.log(markup! {{PrintGitHubDiagnostic::simple(&diagnostic)}});
+        }
+    }
+
```

Comment:
The `diagnostics_to_print` diagnostics are already filtered with the `should_report_to_terminal`, so it's unnecessary here.

Also, there's no need for a second loop, we can already use the loop we have at line 571

Reply:
I did a second loop, so the workflow commands are printed after all the human-readable results.
Of course, we can just emit the commands after each diagnostic.
Did that in 49651245ea.

Reply:
Oh, I see. Do you think it makes a difference? 🤔 I suppose we'll have to experiment and see?

Reply:
Both get printed to different outputs (stderr on diagnositcs vs stdout for workflow commands). When printed on a terminal, this might cause broken error messages because the terminal renders both at the same time (and their buffers bay be flushed at different times). But in CI, it shouldn't make any difference.